src/utils/__tests__/presence-test.js

```diff
@@ -98,18 +100,33 @@ describe('presenceToHumanTime', () => {
           aggregated: { client: 'website', status: 'active', timestamp: currentTimestamp - 100 },
         },
         { away: false, status_text: null, status_emoji: null },
+        eg.recentZulipFeatureLevel,
       ),
     ).toBe('now');
   });
 
-  test('if less than a day, and the user is "away" the user is ...', () => {
+  // TODO(server-6.0): Remove
+  test('Pre-FL 148: if less than a day and user is "away", use imprecise "today"', () => {
     expect(
       presenceToHumanTime(
         { aggregated: { client: 'website', status: 'active', timestamp: currentTimestamp - 100 } },
         { away: true, status_text: null, status_emoji: null },
+        147,
       ),
     ).toBe('today');
   });
+
+  // TODO(server-6.0): Remove; this is the natural behavior and won't need
+  //   testing when the 148 condition is gone.
```

Comment:
Hmm, "this won't need testing, it's the natural behavior" sounds like famous last words :wink: I'd be hesitant to drop a test case on those grounds.

I think this test will end up being removable, but the reason is: when we delete that 148 condition, it will also let us stop passing a `UserStatus` to this function at all. When we do that, this test case will be testing the exact same input as one of the cases above.

Hmm, that fact we can stop passing a `UserStatus` is probably worth a reminder in that other TODO-server comment, though.